crates/egui_demo_lib/src/demo/plot_demo.rs

```diff
@@ -520,37 +516,35 @@ impl LinkedAxisDemo {
             ui.checkbox(&mut self.link_x, "X");
             ui.checkbox(&mut self.link_y, "Y");
         });
-        self.group.set_link_x(self.link_x);
-        self.group.set_link_y(self.link_y);
         ui.horizontal(|ui| {
             ui.label("Linked cursors:");
             ui.checkbox(&mut self.link_cursor_x, "X");
             ui.checkbox(&mut self.link_cursor_y, "Y");
         });
-        self.cursor_group.set_link_x(self.link_cursor_x);
-        self.cursor_group.set_link_y(self.link_cursor_y);
+
+        const LINK_GROUP_NAME: &str = "linked_demo";
```

Comment:
Better to use a scoped `Id` than a global one:

```suggestion
        let link_group_id = ui.id().with("linked_demo");
```